geometry_msgs/Point goal_position
---
bool success
---
geometry_msgs/Point current_position
float32 distance_to_goal